products/red-hat-openshift.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Red Hat OpenShift
+category: server-app
+iconSlug: redhatopenshift
+permalink: /red-hat-openshift
+versionCommand: oc version
+releasePolicyLink: https://access.redhat.com/support/policy/updates/openshift
+releaseImage: https://access.redhat.com/sites/default/files/styles/XL%20-%20Extra%20Large/public/images/ocp_lifecycle_eus_v5.png
+changelogTemplate: https://docs.openshift.com/container-platform/__RELEASE_CYCLE__/release_notes/ocp-{{"__RELEASE_CYCLE__"| replace:'.','-'}}-release-notes.html
+releaseDateColumn: true
+activeSupportColumn: true
+extendedSupportColumn: true
+eolColumn: Maintenance Support
+releases:
+-   releaseCycle: "4.12"
+    releaseDate: 2023-01-17
+    support: true
+    eol: 2024-07-17
+    latest: "4.12.6"
```

Comment:
Is there a way to automatically track new releases here? (We support a variety of [automation](https://github.com/endoflife-date/endoflife.date/wiki/Automation)), but can write our own scraper if needed.

Reply:
I'll check it out

Reply:
I think as long as quay.io isn't supported it won't be possible to fetch these automatically. I don't know of any github project we could track :(

Reply:
They publish the release images here: https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags&tag=latest